src/App.vue

```diff
@@ -60,7 +60,7 @@ const options = {
     minScale: 0.1,
     scaleSpeed: 1.2
   },
-  toolbox: toolboxString,
+  toolbox: toolbox,
```

Comment:
Would removing this line change anything?

Reply:
it would change everything since thats what hands blockly the toolbox

Reply:
Okay